grails-app/assets/javascripts/species.show.js

```diff
@@ -265,8 +265,10 @@ function loadAusTraits() {
     });
 
     $.ajax({url:SHOW_CONF.ausTraitsCountUrl}).done(function (data) {
-        if (data[0] && data[0].explanation){
-            $('#traits-description').html("<span>"+data[0].explanation+"</span>")
+        if (data[0] && data[0]["summary"] && data[0]["AusTraits"] && data[0]["taxon"] ){
```

Comment:
I think this should be implemented with i18n and passing the `data[]` vars via i18n args. This code is used by the Living Atlas community and they require all text to be i18n-ised.

Reply:
As per discussion this morning. Leaving this as is - as it is config controlled to just the Traits tab and  not indented for usage for living-atlases.